Tests/test_imageops.py

```diff
@@ -311,3 +313,30 @@ def autocontrast(cutoff):
 
         assert autocontrast(10) == autocontrast((10, 10))
         assert autocontrast(10) != autocontrast((1, 10))
+
+
+def test_autocontrast_mask():
+    # Test the mask argument of autocontrast
+    with Image.open("Tests/images/bw_gradient.png") as img:
+
+        rect_mask = Image.new("L", img.size, 0)
+        draw = ImageDraw.Draw(rect_mask)
+        x0, y0 = img.size[0]//4, img.size[1]//4
+        x1, y1 = 3*img.size[0]//4, 3*img.size[1]//4
+        draw.rectangle((x0, y0, x1, y1), fill=255)
+
+        assert ImageOps.autocontrast(img, mask=rect_mask) != ImageOps.autocontrast(img)
+
+    # Test the autocontrast with a rectangular mask
```

Comment:
Please could you split this into a second test `def`? That way we know this part doesn't depend on something happening on the previous part.

Reply:
Makes sense. I have split it in new PR.